src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -132,6 +132,11 @@ class NoTemplatesServiceApi(object):
         and trailing
         whitespace.
 
+        Example:
+          >>> from google.example.v1 import no_path_templates_pb2
+          >>> api = NoTemplatesServiceApi()
```

Comment:
Note we use PythonImportHandler for generating method samples for discovery sample gen. The PythonImportHandler has a nontrivial mechanism for avoiding name conflicts, which is always a possibility (however unlikely) even for single-method samples.